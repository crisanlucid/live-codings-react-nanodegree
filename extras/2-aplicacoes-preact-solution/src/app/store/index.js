import { createStore } from 'redux'

const initialState = {
  todos: [
    {
      id: 1,
      text: 'Lorem',
      isCompleted: true
    },
    {
      id: 2,
      text: 'Ipsum',
      isCompleted: false
    },
    {
      id: 3,
      text: 'Dolor',
      isCompleted: true
    }
  ]
}

function reducer (state = {}, action = {}) {
  return state
}

const store = createStore(reducer, initialState)

export default store
