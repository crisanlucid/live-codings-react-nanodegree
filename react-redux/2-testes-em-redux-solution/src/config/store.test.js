import store from './store';

test('store should match snapshot', () => {
  const state = store.getState();
  expect(state).toMatchSnapshot();
});
