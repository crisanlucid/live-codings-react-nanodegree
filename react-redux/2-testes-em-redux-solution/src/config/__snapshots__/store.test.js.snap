// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store should match snapshot 1`] = `
Object {
  "error": false,
  "fetching": false,
  "games": Array [],
}
`;
